src/type/introspection.js

```diff
@@ -221,6 +222,20 @@ export const __Type = new GraphQLObjectType({
         resolve: obj =>
           obj.description !== undefined ? obj.description : undefined,
       },
+      specifiedBy: {
+        type: GraphQLString,
+        resolve: type => {
+          if (!isScalarType(type) || !type.astNode) {
+            return null;
+          }
+
+          const specified = getDirectiveValues(
+            GraphQLSpecifiedDirective,
+            type.astNode,
+          );
+          return specified && specified.by;
```

Comment:
@m14t You shouldn't do that, `astNode` should be used only for error reporting.
Please replicate how `@deprecate` is converted into `deprecationReason`.